git_hooks/hooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+[ -n "$THOTH_NO_PRECOMMIT_TEST" ] && exit 0
+
+which pipenv > /dev/null 2>&1 || {
+RED='\033[0;31m'
+NC='\033[0m'
+echo "${RED}Skipping the pre-commit test,as PIPENV is NOT installed${NC}"
+exit 0
+}
+
+pipenv install --dev
```

Comment:
I think this should be removed and mentioned as a prerequisite, if we install on each commit it will take a long time...

Reply:
yes, we can remove it and keep it as a prerequisite.
only end case: if anyone tries to workon or uninstalls any package from pipenv.